docs/advanced-topics/negative-mining.md

```diff
@@ -0,0 +1,86 @@
+(negative-mining)=
```

Comment:
It took me a while and some googling to understand what this document is about. I think I should step away from reviewing this document and take a stab at rewriting some of it to be clearer in a separate ticket.

Reply:
TODO: 
- Negative Minining is mentioned but not explained
- hard to understand for someone new to the field
-